Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -908,7 +908,7 @@ final class HTTPClientTests: BaseHTTPClientTests<MockETagManager> {
             }
         }
 
-        self.waitForExpectations(timeout: 1)
+        self.waitForExpectations(timeout: defaultTimeout.seconds * TimeInterval(serialRequests))
```

Comment:
I guess this will increase the timeout a lot right? (up to 20 seconds). Just want to confirm we want to do that.

Reply:
Not needed anymore thanks to #2802